### PR TITLE
Opens another viro slot

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request
I had this open, but it went stale with zero response. 
Opens an extra viro slot. Our maps are made with two virologists in mind, anyhow

## Why It's Good For The Game
I didnt go in depth on this in my last PR, so ill sum it up further
Virology is one of the only one slot jobs. Discounting heads, this includes 3 flavor jobs (detective, chaplain, curator) and the Warden. 
Virology is a job with a high entry barrier at the moment. It's a bit intimidating to get into, and I hope having two slots will allow a more experienced virologist to teach a less experienced one.
Having two virology slots also opens up the job to more roleplay after symptom gathering is done. Being able to collaberate and test diseases together, bounce ideas off of each other in character, and show each other tricks is interesting.
Finally, this opens up interesting opportunities for miscommunication, especially as far as virus tests go. And the crew no longer always knows which person exactly is responsible for the deadly plague sweeping the station

## Changelog
:cl:
tweak: adds another viro slot
/:cl:

### CODE REVIEW

File: code/modules/jobs/job_types/virologist.dm [hunk 1]
```diff
@@ -4,8 +4,8 @@
 	department_head = list("Chief Medical Officer")
 	department_flag = MEDSCI
 	faction = "Station"
-	total_positions = 1
-	spawn_positions = 1
+	total_positions = 2
+	spawn_positions = 2
```
Comment (Ivniinvi) [inline code/modules/jobs/job_types/virologist.dm]:
```suggestion
	spawn_positions = 2
```
weird hanging space?

**Review by PowerfulBacon — DISMISSED** [review]
Pretty sure this needs to be set in the jobs.txt config file

**Comment by zeskorion** [comment]
> Pretty sure this needs to be set in the jobs.txt config file

According to crossed, that file is no longer in use

**Comment by PowerfulBacon** [comment]
What? Really?

**Comment by zeskorion** [comment]
> What? Really?

Yep. I had a pr up for that and crossed closed it due to jobs.txt not being in use?

**Pull request merged**